package.json

```diff
@@ -69,5 +70,8 @@
     "snapshotSerializers": [
       "enzyme-to-json/serializer"
     ]
+  },
+  "now": {
+    "alias": "styled-components-website"
```

Comment:
This should be `styled-components.com`, right?

Reply:
Not sure. In my last project final url generated by Now was `<alias>.now.sh`